authorization-api-1_0.md

```diff
@@ -599,7 +602,184 @@ X-Request-ID: bfe9eb29-ab87-4ca3-be83-a1d5d8305720
 ~~~
 {: #example-search-response title="Example of an Search Response"}
 
+## Subject Search API
+The Access Subject Search API does the reverse of the Search API: it enables a PEP or client to find out all the principals that can access a given asset.
```

Comment:
Do you want to rename the existing "Search API" to be "Resource Search API" (and this one is the "Subject Search API")?

Reply:
Done in latest commit.